test/compare/custom/negative_indent-config.json

```diff
@@ -0,0 +1,8 @@
+{
+  "indent": {
+    // yes, you can set negative values. "0" is the real noop.
```

Comment:
None that I could think of so far but it was a good way to test the
negative value on the rocambe-indent helpers

To align the comments we will need a way to decrease the indent (of avoid
adding it)